spec/rspec/matchers/expecteds_for_multiple_diffs_spec.rb

```diff
@@ -0,0 +1,82 @@
+module RSpec
+  module Matchers
+    RSpec.describe ExpectedsForMultipleDiffs do
+
+      # Setup fake differ
+      let(:differ_factory) { double("Differ factory method", :call => differ) }
+      let(:differ) { double("Differ") }
+
+      before do
+        allow(differ).to receive(:diff) do |_actual, expected|
+          [_actual, expected].inspect
+        end
```

Comment:
Given that this isn't a canned response but instead has logic, do you think it would make more sense to just make a fake differ?

``` ruby
class FakeDiffer
  def self.diff(_actual, expected)
    [_actual, expected].inspect
  end
end
```

Then there's no need to re-stub it for each spec.